### ISSUE TITLE
auto-correction for simple spelling mistakes as in word and other tools

### ISSUE THREAD

**Issue by krishvk** [streetsidesoftware/vscode-spell-checker#761]
The extension is great. Can we have an auto-correction feature as we have in MS Word where 'hte' will be auto corrected to 'the' and so on

**Comment by bboysnick5** [comment]
Any progress on this? Mistyping is very common if you do a lot of markdowns.

**Comment by dimateos** [comment]
Yeah having to go from typo to typo is a bit tedious. Something as suggested in #237:

> I've recently discovered https://marketplace.visualstudio.com/items?itemName=sygene.auto-correct. It would be nice if these two extensions could merge with one another other so that when you fix a spelling error, it can add it to a map of words that will be auto-corrected. For example, say you mispell horse as horrse but then utilize this extension to fix the spelling: From that point on, horrse will be autocorrected to horse.

Or maybe a simpler way to automatically correct all words to the first suggestion with a command, probably with some kind of log afterwards. The best idea I have is to add the top suggestion in the error/warning notice instead of "Unknown word."

**So you could go through the list and if they make sense just automatically fix all of them.**
This way I would not need a log, but it maybe important to integrate undo in case something goes wrong.

As an example right now there is the command `cSpell.addIssuesToDictionary`
